crates/matrix-sdk-store-encryption/src/lib.rs

```diff
@@ -674,7 +674,43 @@ impl StoreCipher {
         self.decrypt_value_typed(value.try_into()?)
     }
 
-    /// Decrypt a value after it was fetchetd from the key/value store.
+    /// Decrypt a base64-encoded value after it was fetched from the key/value
+    /// store.
+    ///
+    /// A value can be encrypted using the
+    /// [`StoreCipher::encrypt_value_base64_data()`] method.
+    ///
+    /// # Arguments
+    ///
+    /// * `value` - The EncryptedValueBase64 of a value that should be
+    ///   decrypted.
+    ///
+    /// The method will return the raw decrypted value
+    ///
+    /// # Examples
+    ///
+    /// ```
```

Comment:
You can use `rust,no_run` if you still want to assume it's Rust code, but you don't want to run it (only check it).

Reply:
Thanks, added to all in the file. 885d85740c